org.eclipse.xtext.util/src/org/eclipse/xtext/util/MergeableManifest.java

```diff
@@ -56,6 +70,22 @@
 	@Deprecated
 	private static final String LINEBREAK = Strings.newLine();
 
+	/**
+	 * Adds line breaks to enforce a maximum 72 bytes per line.
+	 */
+	static void make72Safe(StringBuffer line) {
```

Comment:
Looks like we had the intention to use make512Safe at most locations. Should we adjust the remaining pieces, too?

Reply:
Dead code. Will delete.